packages/github-actions/teams-toolkit-cicd/src/main.ts

```diff
@@ -1,19 +1,76 @@
 import * as core from '@actions/core'
-import {wait} from './wait'
+import {ActionInputs} from './constant'
+import {Operations} from './operations'
+import {OperationType} from './enums/operationTypes'
+import {WordsToList} from './utils/words-to-list'
+import {InputsError} from './errors'
+import {Capability} from './enums/capabilities'
+import {BaseError} from './base-error'
 
 async function run(): Promise<void> {
   try {
-    const ms: string = core.getInput('milliseconds')
-    core.debug(`Waiting ${ms} milliseconds ...`) // debug is only output if you set the secret `ACTIONS_RUNNER_DEBUG` to true
+    let projectRoot = core.getInput(ActionInputs.ProjectRoot)
```

Comment:
NIT: maybe
```
const projectRoot = core.getInput(ActionInputs.ProjectRoot) ?? process.env.GITHUB_WORKSPACE
```

Reply:
True. I tried to use this simpler style, but it didn't work for the case when getInput returns empty string. So keep it for now.